### DISCUSSION TITLE
cannot Choose theme for my page

### DISCUSSION THREAD

**Discussion by Zoueng** [community/community#46500] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Hello everybody!
I would like to create a page on GitHub.
I follow the Official Tutorial step by step. 
After creating a new repository,I need to choose a theme for my page.
But I was unable to find the theme chooser on the "GitHub Pages"
is the difference between my interface and the one shown in the official tutorial: 
![mine](https://user-images.githubusercontent.com/124218205/216985101-db338bd3-f54e-45ae-9ac3-a05cf44f1b39.png)
![offical](https://user-images.githubusercontent.com/124218205/216985137-f710c75c-06c3-44a8-82f4-c60d439a3cc1.png)
Please tell me what to do next, Thank you!

**Comment by yoannchaudet** [comment]
The marketing site pages.github.com is getting revamped but it is taking longer than expecting.

The theme chooser has been deprecated for security reasons. You can just as easily add a theme following the documentation (which is current): https://docs.github.com/en/pages/setting-up-a-github-pages-site-with-jekyll/adding-a-theme-to-your-github-pages-site-using-jekyll.
